diff --git a/CMakeLists.txt b/CMakeLists.txt
index 47bbaa15d..f04bbae88 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -155,17 +155,8 @@ endif()
 
 mapnik_find_package(Freetype REQUIRED)
 
-# try to find harfbuzz with the native configuration and fallback to our "own" FindHarfBuzz
-mapnik_find_package(harfbuzz CONFIG QUIET)
-if(harfbuzz_FOUND)
-    message(STATUS "Found harfbuzz native cmake")
-    list(APPEND MAPNIK_OPTIONAL_LIBS harfbuzz::harfbuzz)
-else()
-    # we use our "own" FindHarfBuzz. See https://github.com/mapnik/mapnik/pull/4191#issuecomment-874728157 for more details
-    message(STATUS "Fallback to FindHarfBuzz")
-    mapnik_find_package(HarfBuzz ${HARFBUZZ_MIN_VERSION} REQUIRED COMPONENTS ICU)
-    list(APPEND MAPNIK_OPTIONAL_LIBS HarfBuzz::HarfBuzz HarfBuzz::ICU)
-endif()
+pkg_check_modules(harfbuzz REQUIRED IMPORTED_TARGET harfbuzz)
+list(APPEND MAPNIK_OPTIONAL_LIBS PkgConfig::harfbuzz)
 
 if(USE_EXTERNAL_MAPBOX_GEOMETRY)
     # this is used to provide a way to specify include dirs with CACHE VARIABLES
